transcom/mymove
# B 20997 int add pptas under prime simulator

## [B-20997](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20997)

## Summary

> [!IMPORTANT]  
> This update is not for production release. This is strictly to allow PPTAS to be testable with the Prime Simulator in the integration/staging environments.

Adds the PPTAS API to be able to be called using the Prime Simulator Routing (so that Postman may be used to call the endpoints contained).
## Verification Steps for Reviewers
1. Run MilMove locally
2. log in to prime simulator, make sure that cookies are syncing with Postman
3. Make a call to the endpoint using the "list moves" while run locally in postman using `http://officelocal:3000/pptas/v1/moves?since=2021-07-23T18:30:47.116Z`
4. Data returned does not matter, only that the error is not 404 not found. Any response aside from that would be passing.
